test-infra/README_accep_test.md

```diff
@@ -114,6 +115,7 @@ Passing an environment value of "yes" to the skip parameter allows you to skip t
 | aviatrix_version                     | SKIP_VERSION                       |                                                                                |
 | aviatrix_vgw_conn                    | SKIP_VGW_CONN                      | aviatrix_gateway + AWS_BGP_VGW_ID                                              |
 | aviatrix_vpc                         | SKIP_VPC                           | aviatrix_account                                                               |
+| aviatrix_vpn_cert_download           | SKIP_VPN_CERT_DOWNLOAD             | aviatrix_vpn_user + aviatrix_saml_endpoint                                     |
```

Comment:
Need to update test-infra/cmdExportOutput.sh as well

Reply:
Updated